### ISSUE TITLE
Gridseed Timeout command

### ISSUE THREAD

**Issue by ghost** [luke-jr/bfgminer#612]
Is there a gridseed timeout command where if last share time is x amount of time then restart or reshare/rework or do something with the miner?

Thank you :)

**Comment by luke-jr** [comment]
No, because "last share" is not a reasonable metric to use. --cmd-idle should execute if the device stops working, though...

**Comment by ghost** [comment]
Ok cool would you think to run --cmd-dead (or --cmd-sick) instead or idle should be fine?

Also what would the command be to restart the miner when the event is triggered

**Comment by luke-jr** [comment]
Depends on if BFGMiner is flagging the device as dead or sick. You'd need to setup the RPC API to do a clean restart I think.
